spring-cloud-kubernetes-integration-tests/istio/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/base.xml" />
+    <logger name="org.hibernate.validator" level="info" /> <!-- Validator prints a lot of debug messages during integration tests -->
```

Comment:
This can be done in application.properties

Reply:
But would it apply to test as well (which keep in mind does not involve Spring)?

Reply:
@spencergibb I will remove this one then.

Reply:
@geoand are you ok with removing that one?

Reply:
Can't do much harm either ways :)